handlers/delete_session.go

```diff
@@ -0,0 +1,31 @@
+package handlers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/franela/play-with-docker/services"
+	"github.com/gorilla/mux"
+)
+
+func DeleteSession(rw http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	sessionId := vars["sessionId"]
+	log.Println(sessionId)
+
+	session := services.GetSession(sessionId)
+
+	if session == nil {
+		rw.WriteHeader(http.StatusNotFound)
+		return
+	}
+
+	err := services.CloseSession(session)
+	if err != nil {
```

Comment:
In case of an error we should print the error so then we can try to see what happened when looking into the logs.